### DISCUSSION TITLE
I am a Chinese student !

### DISCUSSION THREAD

**Discussion by ufec** [community/community#17734] [GitHub Education]
I am a Chinese student. I try to apply for GitHub education pack.

But what surprised me was that the process was so difficult.

I tried to upload my report card of the last semester, my student ID card and the screenshot of our school's portal website. The above still can't prove my student identity. Later, I applied for the school's email, and I tried to use the email to apply, which still can't prove my student identity. What should I do? My student ID card is all in Chinese. I have attached instructions: the left contains the school description, and the right contains student information, enrollment information and graduation information. Why do you still refuse? I have tried five times. Please tell me what I should do

**Comment by Anthoooooooony** [comment]
Try not using proxy. Seems that applications will be instantly refused if your IP is too far from your school.

**Reply by ufec** [reply]
Thank you for your reply. I also tried to use campus VPN to ensure my IP source, but it seems that it doesn't help. The official has always prompted me that the uploaded pictures have been modified, but I didn't do that. The school requires students to fill in the information themselves. I can only come according to the school's requirements. My student card contains the name of the school, my basic information and the steel seal of the school. Is it unrecognizable in Chinese? I tried to get in touch with GitHub support, but there was no official reply. What should I do
